backend/apps/vacancies/models.py

```diff
@@ -58,14 +70,28 @@ class Vacancy(models.Model):
         if we need in vacancy show salary
     """
 
-    title = models.CharField(_("Title"), max_length=64, db_index=True)
+    title = models.CharField(
+        _("Title"), 
+        max_length=64, 
+        db_index=True
+    )
     keywords = TaggableManager()
     type_of_employment = models.CharField(
-        _("Type of employment"), choices=EmploymentChoices.choices, db_index=True, max_length=32
+        _("Type of employment"), 
```

Comment:
Employment Type, try to use such clauses, they occupy less space. Stone Wall == Wall of Stone